### ISSUE TITLE
Wrong node version

### ISSUE THREAD

**Issue by tech247gb** [cypress-io/circleci-orb#423]
## The orb does not work as expected

Still its chosing 16 as node version.

version: 2.1
orbs:
  cypress: cypress-io/cypress@3
executors:
    default:
        docker:
            - image: cimg/node:14.17.6-browsers
workflows:
  use-my-orb:
    jobs:
      - cypress/run:
          working-directory: webclient
          cypress-command: npx cypress run --browser edge

**Comment by nagash77** [comment]
Hi @tech247gb , You will need to provide more information and details about what you are expecting and how you are encountering this problem.

**Comment by tech247gb** [comment]
I am expecting npm install happen with node version 14.17.6. But its happening with 16

**Comment by tech247gb** [comment]
tried that but not working as expected 
`version: '2.1'
orbs:
  cypress: cypress-io/cypress@3
jobs:
  install-and-persist:
    executor: cypress/default
    node-version: 14.17.6
    steps:
      - cypress/install
      - persist_to_workspace:
          paths:
            - .cache/Cypress
            - project
          root: ~/
  run-tests-in-parallel:
    executor: cypress/default
    node-version: 14.17.6
    steps:
      - attach_workspace:
          at: ~/
      - cypress/run-tests:
          cypress-command: npx cypress run --component --parallel --record
workflows:
  use-my-orb:
    jobs:
      - install-and-persist:
          name: Install & Persist To Workspace
      - run-tests-in-parallel:
          name: Run Tests in Parallel
          requires:
            - Install & Persist To Workspace`

**Comment by LeviHeberRI** [comment]
@jordanpowell88 cypress/run does not accept an executor argument, so it is also not possible to change the node version.

**Comment by jordanpowell88** [comment]
> @jordanpowell88 cypress/run does not accept an executor argument, so it is also not possible to change the node version.

You are correct. You need to specify a different `executor` than the default being used by the `cypress/run` job. See this comment on how to implement https://github.com/cypress-io/circleci-orb/issues/423#issuecomment-1515217741

**Comment by csaszika** [comment]
@jordanpowell88 cypress/run does not accept an executor argument
